pkg/build/registry/buildconfig/webhook.go

```diff
@@ -94,7 +94,7 @@ func (h *WebHookHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // ProcessWebHook does the actual work of processing the webhook request
 func (w *WebHookHandler) ProcessWebHook(writer http.ResponseWriter, req *http.Request, ctx apirequest.Context, name, subpath string) error {
-	parts := strings.Split(subpath, "/")
+	parts := strings.Split(strings.TrimPrefix(subpath, "/"), "/")
```

Comment:
@bparees fyi

Reply:
@soltysh thanks....  k8s changed something about how the provide the request path to us??

Reply:
Subresource path was stripping leading/trailing slashes incorrectly. Fixed in 1.7.8

Reply:
Yup, just that.